### PR TITLE
Add license key in __openerp__.py

### PR DESCRIPTION
**Pull request by yvaucher**

### CODE REVIEW

**Comment by lepistone** [comment]
:+1:

File: __unported__/barcode_link/__openerp__.py [hunk 1]
```diff
@@ -32,6 +32,7 @@
 
 """,
     "website": "http://www.julius.fr",
+    "license": "AGPL-3",
```
Comment (yvaucher) [inline __unported__/barcode_link/__openerp__.py]: GPL-3

File: __unported__/tr_barcode_config/__openerp__.py [hunk 2]
```diff
@@ -37,6 +37,7 @@ class product_product(barcode_osv.barcode_osv):
 
 """,
     "website": "http://www.julius.fr",
+    "license": "AGPL-3",
```
Comment (yvaucher) [inline __unported__/tr_barcode_config/__openerp__.py]: GPL-3

File: __unported__/tr_barcode_field/__openerp__.py [hunk 3]
```diff
@@ -31,6 +31,7 @@
 
 """,
     "website": "http://www.julius.fr",
+    "license": "AGPL-3",
```
Comment (yvaucher) [inline __unported__/tr_barcode_field/__openerp__.py]: GPL-3

File: __unported__/tr_barcode_on_picking/__openerp__.py [hunk 4]
```diff
@@ -24,6 +24,7 @@
     "version": "1.1",
     "author": "Julius Network Solutions",
     "website": "http://www.julius.fr",
+    "license": "AGPL-3",
```
Comment (yvaucher) [inline __unported__/tr_barcode_on_picking/__openerp__.py]: GPL-3

File: __unported__/tr_barcode_on_prodlots/__openerp__.py [hunk 5]
```diff
@@ -24,6 +24,7 @@
     "version": "1.1",
     "author": "Julius Network Solutions",
     "website": "http://www.julius.fr",
+    "license": "AGPL-3",
```
Comment (yvaucher) [inline __unported__/tr_barcode_on_prodlots/__openerp__.py]: GPL-3

File: __unported__/tr_barcode_on_product/__openerp__.py [hunk 6]
```diff
@@ -24,6 +24,7 @@
     "version": "1.1",
     "author": "Julius Network Solutions",
     "website": "http://www.julius.fr",
+    "license": "AGPL-3",
```
Comment (yvaucher) [inline __unported__/tr_barcode_on_product/__openerp__.py]: GPL-3

File: __unported__/tr_barcode_on_tracking/__openerp__.py [hunk 7]
```diff
@@ -24,6 +24,7 @@
     "version": "1.1",
     "author": "Julius Network Solutions",
     "website": "http://www.julius.fr",
+    "license": "AGPL-3",
```
Comment (yvaucher) [inline __unported__/tr_barcode_on_tracking/__openerp__.py]: GPL-3

**Comment by StefanRijnhart** [comment]
:+1:

**Comment by pedrobaeza** [comment]
:+1:

**Pull request merged**